cmd/git/status.go

```diff
@@ -0,0 +1,41 @@
+package git
+
+import (
+  "fmt"
+  "os"
+
+  "github.com/Originate/gt/cmd/util"
+)
+
+
+func EnsureDoesNotHaveConflicts() {
+  if HasConflicts() {
+    util.ExitWithErrorMessage("You must resolve the conflicts before continuing")
+  }
+}
+
+
+func GetRootDirectory() string {
+  return util.GetCommandOutput([]string{"git", "rev-parse", "--show-toplevel"})
```

Comment:
What do you think about making `util.GetCommandOutput` a [variadic function](https://gobyexample.com/variadic-functions)? It would be used like this:

```go
return util.GetCommandOutput("git", "rev-parse", "--show-toplevel")